backend/tests/unit/oasst_backend/README.md

```diff
@@ -0,0 +1,5 @@
+# Backend Testing Guidelines
+
+## Testing Endpoints
+
+TBD
```

Comment:
If possible could you provide some basic doc so others ones have a starting point? ty :)